### ISSUE TITLE
Guidance on making a screencast

### ISSUE THREAD

**Issue by Uran198** [coala-analyzer/engagement#8]
Maybe, it would be useful to have somewhere around here some guidance, tips and tricks on how to make screencasts.

**Comment by sils1297** [comment]
Yes, see also https://github.com/hypothesist/termtype/blob/master/coala_intro.example

**Comment by sils1297** [comment]
@Uran198 do you want this documented in the repo somewhere?

**Comment by Uran198** [comment]
I don't think it's important where it is until it's easy to find. Maybe, gathering all engagement related stuff in this repo can make it simpler to find. Like pointing to one repo and saying that there is all you need related to engagement.

**Comment by sils1297** [comment]
I think this should be done upstream as termtype documentation, CC @hypothesist
